test/DynamoCoreWpfTests/NodeAutoCompleteSearchTests.cs

```diff
@@ -164,35 +164,82 @@ public void NodeSuggestions_InputPortBuiltInNode_AreCorrect()
             Assert.AreEqual("string", type);
 
             var searchViewModel = (ViewModel.CurrentSpaceViewModel.NodeAutoCompleteSearchViewModel as NodeAutoCompleteSearchViewModel);
-            searchViewModel.PortViewModel = inPorts[1];
+            searchViewModel.PortViewModel = inPorts[0];
```

Comment:
I don't understand this change.

Reply:
It looks like the inPorts[0] corresponds to a `var[]..[]` type, which should be skipped, correct? In which case, shouldn't it return 0 results?

Reply:
`var[]..[]` is not in the skipped type list. `var` is skipped as of now.
Do you want to add `var[]..[]` as well?

Reply:
I think @mjkkirschner made a change that strips the array rank from the type. But in any case, I think you should strip the rank and skip just the type. That way it will skip all rank variations, meaning var[], var[][], etc.